yaqd-core/yaqd_core/_has_transformed_position.py

```diff
@@ -83,17 +96,16 @@ def set_native_reference(self, native_position):
         self._state["native_reference_position"] = native_position
 
     def set_native_position(self, native_position):
-        self.set_position(self.to_transformed(native_position))
+        super().set_position(native_position)
 
     def get_native_position(self) -> float:
-        return self.to_native(super().get_position())
+        return self._state["position"]
 
     def get_native_destination(self) -> float:
         return self.to_native(super().get_destination())
```

Comment:
Shouldn't self._state["destination"] also be changed to native units upon inheiritance of HasTransformedPosition?

Reply:
good catch--get_native_destination was not working correctly.  get_native_destination now pulls directly from state